lua/ttt2/extensions/cvars.lua

```diff
@@ -0,0 +1,174 @@
+---
+-- cvars extensions
+-- @author ZenBreaker
+-- @module cvars
+
+if SERVER then
+	AddCSLuaFile()
+end
+
+local cvars = cvars
+
+local messageIdentifier = -1
+local identityBitCount = 8
+local maxUInt = 2 ^ identityBitCount
+
+local serverConVars = {}
+local functionCache = {}
+
+local function IsAdmin (ply)
```

Comment:
I don't see the necessity for this function

Reply:
It is not necessary. But we didnt know which rights we wanted to give, so I let that be easily changeable.
As we kinda discussed it should be superadmin, I can delete it.